ZenPacks/zenoss/ZenPackLib/tests/test_zen_25315.py

```diff
@@ -45,25 +45,35 @@
             parser: Auto
 """
 
+def win_shell_installed():
+    '''Return True if Duration Threshold is installed'''
```

Comment:
This doc string is confusing since you're loading Windows shell dataource.

Also, you should use double quotes here for the docstring